addons/point_of_sale/models/pos_order.py

```diff
@@ -1111,6 +1111,7 @@ def _export_for_ui(self, order):
             'state': order.state,
             'account_move': order.account_move.id,
             'id': order.id,
+            'server_id': order.id,
```

Comment:
Was handled in the frontend 
``this.server_id = json.server_id || json.id || false;``

Reply:
oh, it means that during initialization of the order in the frontend, the "id" is used as server_id, and we don't need to put "server_id" in the export for ui. Maybe we should do same logic for other pos frontend models.

Reply:
I'll remove and update the init_from_JSON of the pos ui models.